contracts/src/contracts/Teleporter/AvalancheICTTRouterFixedFees.sol

```diff
@@ -179,12 +247,23 @@ contract AvalancheICTTRouterFixedFees is
         address bridgeSource = tokenToSourceBridge[address(0)];
         DestinationBridge memory destinationBridge =
             tokenDestinationChainToDestinationBridge[destinationChainID][address(0)];
+        MinBridgeFees memory minBridgeFees =
+            destinationChainTokenToMinBridgeFees[destinationChainID][address(0)];
 
         uint256 primaryFeeAmount = (msg.value * primaryRelayerFeeBips) / BASIS_POINTS_DIVIDER;
         uint256 secondaryFeeAmount = destinationBridge.isMultihop
             ? (msg.value * secondaryRelayerFeeBips) / BASIS_POINTS_DIVIDER
             : 0;
```

Comment:
If the check on `isMultihop` is done at bridge registration, we don't have to rely on it here.

Reply:
Nvm this is wrong